Kinvey/KinveyTests/PerformanceProductTestCase.swift

```diff
@@ -446,7 +444,6 @@ class PerformanceProductTestCase: KinveyTestCase {
     func testPerformance_SyncStore_ConsideringNetworkLatency_10K() {
         let query = Query()
         query.skip = 0
-        // TODO: change after https://kinvey.atlassian.net/browse/BACK-2315 gets solved
         query.limit = 10000 - 1
```

Comment:
This `TODO` was removed, but was the change made? It looks like `BACK-2315` has been resolved.